web-registry/src/components/organisms/CreditBatches/CreditBatches.tsx

```diff
@@ -108,6 +108,12 @@ const CreditBatches: React.FC<CreditBatchProps> = ({
     );
   }
 
+  const someTx = batches.some(b => b.txhash);
+
+  if (!someTx) {
+    columnsToShow = columnsToShow.filter(col => col.id !== 'txhash');
```

Comment:
Do we have a variable we can use for `txhash`? Just in case the column name change, the filter will still work this way.